lib/lotus/commands/new.rb

```diff
@@ -27,6 +27,14 @@ def initialize(app_name, environment, cli)
       def start
         @command.start
       end
+
+      private
+      def _get_real_app_name(app_name)
+        raise ArgumentError.new("Invalid application name. If you want to set application path, please use --path option") if app_name.include?('/')
```

Comment:
I think is better put this because it's a long line

```ruby
if app_name.include?('/')
  raise ArgumentError.new("Invalid application name. If you want to set application path, please use --path option")
end
```